services/registry.dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM mhart/alpine-node:12
```

Comment:
Working from one directory above `/registry` because we can't add files from outside the Docker context (the `common` directory).